src/components/TodoApp.tsx

```diff
@@ -0,0 +1,81 @@
+import { useEffect, useState } from 'react';
+
+import { Todo } from '../types/Todo';
+import { Notification } from '../types/Notification';
+import { Status } from '../types/Status';
+
+import * as TodosService from '../api/todos';
+
+import { UserWarning } from '../UserWarning';
+import { TodoHeader } from './todoHeader';
+import { TodoFooter } from './TodoFoter';
+import { TodoList } from './TodoList';
+import { TodoNotificaition } from './TodoNotification';
+
+const USER_ID = 11293;
+
+export const TodoApp = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState<Status>(Status.ALL);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  useEffect(() => {
+    TodosService.getTodos(USER_ID)
+      .then(setTodos)
+      .catch(() => {
+        setErrorMessage(Notification.load);
+      });
+  }, []);
+
+  useEffect(() => {
+    if (errorMessage) {
+      setTimeout(() => {
+        setErrorMessage('');
+      }, 3000);
+    }
+  }, [errorMessage]);
+
+  const filteredTodos = todos.filter(todo => {
+    switch (status) {
+      case Status.ACTIVE:
+        return todo.completed === false;
+
+      case Status.COMPLETED:
+        return todo.completed === true;
+
+      default:
+        return true;
+    }
+  });
```

Comment:
You still filter in any case

add helper function
```
  const getVisibleTodos = (todos, status) => {
    switch (status) {
      case Status.ACTIVE:
        return todos.filter(todo => !todo.completed);

      case Status.COMPLETED:
        return todos.filter(todo => todo.completed);

      default:
        return todos;
    }
  }
```

```suggestion

  const filteredTodos = getVisibleTodos(todos, status);
```